rpcs3/Emu/SysCalls/Modules/cellNetCtl.cpp

```diff
@@ -48,7 +52,15 @@ int cellNetCtlGetState(vm::ptr<u32> state)
 {
 	cellNetCtl.Warning("cellNetCtlGetState(state_addr=0x%x)", state.addr());
 
-	*state = CELL_NET_CTL_STATE_Disconnected; // TODO: Allow other states
+	// TODO: Unix/Linux and Mac implementation
+#ifdef _WIN32
+	if (InternetCheckConnection(L"http://www.google.com", FLAG_ICC_FORCE_CONNECTION, 0))
```

Comment:
I missed a word there, I wanted to write "should not directly effect the emulated system"